<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewEditSingleThreshDlg</class>
 <widget class="QDialog" name="NewEditSingleThreshDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget
{
	border-image:none;
}

QWidget#widget
{
	border-image: url(:/images/images/whole_back_2.png);
}

QLabel
{
	color: rgb(255, 255, 255);
}

QPushButton
{
	width:80;
	height:27;
	color: rgb(255, 255, 255);
	border-radius: 8px;  
	
	border-image: url(:/images/images/button_search.png);
}

QLineEdit
{
	color: rgb(0, 0, 0);
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
        <item>
         <widget class="QLabel" name="switchnameLabel">
          <property name="text">
           <string>交换机名称</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="switchName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
        <item>
         <widget class="QLabel" name="switchportLabel">
          <property name="text">
           <string>端口</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QSpinBox" name="switchPort"/>
          </item>
          <item>
           <widget class="QPushButton" name="add">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="reduce">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2">
        <item>
         <widget class="QLabel" name="typeLabel">
          <property name="text">
           <string>类型</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="type">
          <item>
           <property name="text">
            <string>进速率</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>出速率</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,2">
        <item>
         <widget class="QLabel" name="valueLabel">
          <property name="text">
           <string>阈值</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="value"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="okButton">
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton">
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
